graphql_vuestorefront/models/product.py

```diff
@@ -44,18 +56,24 @@ def _compute_public_categ_slug_ids(self):
 
     @api.depends('name')
     def _compute_website_slug(self):
-        langs = self.env['res.lang'].search([])
-
+        """
+            Simplifying the method and removing lang from it.
+            IT First has the same product name for all languages
```

Comment:
I think better not use specific company names in a fork thats kind of still generic:)